alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportGetWorkflowAlertsAction.kt

```diff
@@ -126,13 +126,18 @@ class TransportGetWorkflowAlertsAction @Inject constructor(
                         .field("trigger_name")
                 )
         }
+        // if alert id is mentioned we cannot set "from" field as it may not return id. we would be using it to paginate associated alerts
+        val from = if (getWorkflowAlertsRequest.alertIds.isNullOrEmpty())
+            tableProp.startIndex
+        else 0
```

Comment:
The `startIndex` isn't being passed over to `getAssociatedAlerts`

Reply:
we dont need it to be passed over. we will use `tableProp.startIndex`

we need this check on `from` param if alertId is mentioned because we cannot paginate for single id in search request to fetch the chained alerts